src/test/java/com/ably/kafka/connect/integration/ChannelSinkTaskTest.java

```diff
@@ -0,0 +1,295 @@
+/*
+  Copyright © 2021 Ably Real-time Ltd. (support@ably.com)
```

Comment:
I recognise you've inherited this header from the code you've moved, but we've been avoiding any boilerplate-esque headers in SDK repositories up until now, relying upon the presence of `COPYRIGHT` and `LICENSE` files at repository root to do that work.

Reply:
That's a good idea. I removed Copyright wording from header here https://github.com/ably/kafka-connect-ably/pull/49/commits/aaebb05cade80b55b6d4fefe0ed8aa4b8fcaabf5
I also updated copyright information ab34e5c6a5fcff450d5fe41f408072a28c73a090